packages/contractkit/src/wrappers/BaseWrapper.ts

```diff
@@ -27,17 +30,26 @@ export function toBigNumber(input: string) {
   return new BigNumber(input)
 }
 
+export function toBuffer(input: string | string[]) {
+  return Buffer.from(trimLeading0x(input as string), 'hex')
+}
+
+export function parseBuffer(buf: Buffer) {
+  return '0x' + buf.toString('hex')
+}
+
+type bytes = Array<string | number[]>
+export function parseBytes(input: Buffer | string): bytes {
+  return input as any
+}
+
 /** Parse string -> int */
 export function toNumber(input: string) {
   return parseInt(input, 10)
```

Comment:
I realize you didn't write this but can you make this use BigNumber which is more robust and used for number parsing elsewhere?
cc @mcortesi